### PR TITLE
QA Fixes

### PR DESCRIPTION
**Pull request by CharlesJonah**
1. Move FAQs under About Us
2. Move Get Started at the bottom of the page next to Who We are.
3. Eliminate Get Started from the Top Nav
4. Add two new buttons on the top Nav: I'm An Agent and I'm A Home Buyer
5.Make the footer responsive on small screens

**Note**
The hamburger menu is lacking the animation it had before, I will fix in a later PR

**Demo**
https://www.useloom.com/share/849e9aeb676d4201a1dfc7bfeb996318

### CODE REVIEW

File: blog/src/components/harmburger/harmburger.js [hunk 1]
```diff
@@ -69,7 +69,10 @@ export default class Harmburger extends Component {
             modal += ` ${open}`;
         }
         return (
-            <div className={modal}>
+            <div
+                className={modal}
+                style={this.props.isHarmburgerOpen ? { display: 'block' } : { display: 'none' }}
```
Comment (adongu) [inline blog/src/components/harmburger/harmburger.js]: Hey Charles, its better to toggle styles with `className` to keep styles separated from JS logic. Could you do this with `className` instead?

File: blog/src/components/harmburger/harmburger.js [hunk 2]
```diff
@@ -98,31 +101,37 @@ export default class Harmburger extends Component {
...
                                 <div className={main__menu}>
                                     <div
                                         onClick={this.externalRoutesNavigator.bind(
                                             null,
-                                            `${BASE_URL}signin`,
+                                            `${BASE_URL}signup`,
                                         )}
                                     >
-                                        Login
+                                        I am an Home Buyer
```
Comment (adongu) [inline blog/src/components/harmburger/harmburger.js]: This should be `I am a Home Buyer`. Is the copy saying `I Am an Home Buyer?`

File: blog/src/styles/index.module.scss [hunk 3]
```diff
@@ -124,21 +124,28 @@ body {
...
+    align-items: center;
     font-size: 12px;
     font-weight: bold;
     background-color: #ffee8b;
-    padding: 13px 21px;
     font-family: Poppins, Verdana, Geneva, Tahoma, sans-serif;
     border-radius: 5px;
     text-decoration: none;
-    color: #111111;
+    color: #111;
```
Comment (adongu) [inline blog/src/styles/index.module.scss]: Thanks for updating it to the shorten version!

File: blog/src/components/harmburger/harmburger.js (hunk 2 above)
Comment (CharlesJonah) [reply blog/src/components/harmburger/harmburger.js]: I think it correct

Comment (adongu) [reply blog/src/components/harmburger/harmburger.js]: I think you are still using `an` instead of `a` in the sentence.

Comment (CharlesJonah) [reply blog/src/components/harmburger/harmburger.js]: Sorry for this, I don't know how I was not able to spot that, I've fixed it

**Pull request merged**